app/assets/stylesheets/roboto.scss

```diff
@@ -0,0 +1,80 @@
+/* cyrillic-ext */
```

Comment:
I'm a little confused by this. It's called Roboto but seems to reference Transport.

If these files actually are Transport I'm not sure if we ought to be using them: ["It’s licensed for use on the GOV.UK domains only"](https://govuk-elements.herokuapp.com/typography/) and ["If your service is publicly available on a subdomain other than service.gov.uk, use an alternative typeface like Helvetica or Arial."](https://design-system.service.gov.uk/styles/typography/).

If this is actually Roboto rather than Transport is there a way we can make that more obvious, and avoid unnecessary references to Transport?

Reply:
I am thinking of deleting this file as I can’t find any reason of needing it, unless we need to use this font. Even so I would prefer of adding a reference of a url

Reply:
I think if we want Roboto we can use it from Google Fonts — it's possible to download it from Google Fonts and serve it directly, though, which might have a (very small) advantage in not relying on the third-party service.

I don't think any harm at all would be done by deleting this file and the associated `.woff2` files, since the browser will always use a fallback font. I don't think we need to be referencing Transport anywhere.

Reply:
I agree! Will wait @thatkevin if there’s a reason of adding it, if not I will delete it